pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -200,6 +285,16 @@ spec:
       type: ThirdParty
   {{ end }}`
 
+	clusterWideSMCPWithProfile = `
+  apiVersion: maistra.io/v2
+  kind: ServiceMeshControlPlane
+  metadata:
+	name: {{ .Name }}
+  spec:
+	version: {{ .Version }}
+	profiles:
+  	- clusterScoped`
```

Comment:
How does the operator know which profile to apply if the file with clusterScoped profile is named profile.yaml?

Reply:
because we are using `oc.ApplyTemplate(t, meshNamespace, clusterWideSMCPWithProfile, ossm.DefaultSMCP())` in this command we pass `ossm.DefaultSMCP()` and this includes the default version that is `env.GetSMCPVersion()`. Note: that this test case will not run in smcp version less than 2.4: `MinVersion(version.SMCP_2_4)`

Reply:
I'm not completely sure but checking this example from Marko I think it's: https://issues.redhat.com/browse/OSSM-3249